File: Moring Snacks/JavaScript./Armstrong.js
let number =153
let digits =number
let sumOfCube= 0
while ( digits!=0){
     let seperateDigits = digits%10
     let cube= seperateDigits**3
          sumOfCube += cube
          digits= Math.floor(digits/10)
}
     if (sumOfCube==number){
          console.log("The number is an Armstrong number")
}
     else{
           console.log("The number is not an Armstrong number")
}

File: Morning Snacks/Python/Armstrong.py
number = int(input("Enter a number: "))
digits = number
sumOfCube = 0
numberLength = len(str(number))

while digits != 0:
    seperateDigits = digits % 10
    cube = seperateDigits ** numberLength
    sumOfCube += cube
    digits = digits // 10

if sumOfCube == number:
    print("The number is an Armstrong number")
else:
    print("The number is not an Armstrong number")

